cost-analyzer/templates/cost-analyzer-frontend-config-map-template.yaml

```diff
@@ -320,9 +320,8 @@ data:
         }
     {{- end }}
 
-    # Query Service Replicas (QSR) proxy
     {{- if and (.Values.kubecostDeployment) (.Values.kubecostDeployment.queryServiceReplicas) (gt (.Values.kubecostDeployment.queryServiceReplicas | toString | atoi) 0) }}
-
+        # Query Service Replicas (QSR) proxy
```

Comment:
I just moved the comment inside the if block because it is confusing in the configmap otherwise.